ui/raidboss/data/02-arr/trial/shiva-hm.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+// TODO: should the post-staff "spread" happen unconditionally prior to marker?
+
+[{
+  zoneRegex: {
+    en: /^Akh Afah Amphitheatre \(Hard\)$/,
+  },
+  zoneId: ZoneId.AkhAfahAmphitheatreHard,
+  timelineFile: 'shiva-hm.txt',
+  timelineTriggers: [
+    {
+      id: 'ShivaHm Absolute Zero',
+      regex: /Absolute Zero/,
+      beforeSeconds: 5,
+      condition: Conditions.caresAboutAOE(),
+      // These are usually doubled, so avoid spamming.
+      suppressSeconds: 10,
+      response: Responses.aoe(),
+    },
+    {
+      id: 'ShivaHm Icebrand',
+      regex: /Icebrand/,
+      beforeSeconds: 5,
+      response: Responses.tankCleave(),
+    },
+  ],
+  triggers: [
+    {
+      id: 'ShivaHm Hailstorm Marker',
+      netRegex: NetRegexes.headMarker({ id: '001D' }),
+      condition: Conditions.targetIsYou(),
+      response: Responses.spread('alert'),
+    },
+    {
+      id: 'ShivaHm Glacier Bash',
+      netRegex: NetRegexes.startsUsing({ id: '9A1', capture: false }),
+      response: Responses.getBehind('info'),
+    },
+    {
+      id: 'ShivaHm Diamond Dust',
```

Comment:
Is this being used?

Reply:
Oops, thanks! I was trying to decide if I wanted a shiva circle timing callout like I did on extreme and then decided that was too much work.